R/RLearner_regr_GPfit.R

```diff
@@ -9,9 +9,12 @@ makeRLearner.regr.GPfit = function(){
       makeLogicalLearnerParam(id = "trace", default = FALSE, tunable = FALSE),
       makeIntegerLearnerParam(id = "maxit", default = 100, lower = 1),
       makeUntypedLearnerParam(id = "optim_start", default = NULL),  
-      makeLogicalLearnerParam(id = "scale", default = TRUE)
+      makeLogicalLearnerParam(id = "scale", default = TRUE),
+      makeDiscreteLearnerParam(id = "corr_type", values = c("exponential", "matern"), default = "exponential"),
+      makeIntegerLearnerParam(id = "matern_nu_k", default = 0L, lower = 0L, requires = quote(corr_type == "matern")), 
+      makeNumericLearnerParam(id = "exp_power", default = 1.95, lower = 1.0, upper = 2.0, requires = quote(corr_type == "exponential"))
     ),
```

Comment:
Can you rename `corr_type`, `matern_nu_k` and `exp_power` to the same names as in GPfit. => `type`, `nu` and `power`